### ISSUE TITLE
[DOC]: Set up prettier and pre-commit hook to prettify codebase

### ISSUE THREAD

**Issue by cdrani** [cdrani/chorus#211]
### Implementation:

1. Setup husky: https://typicode.github.io/husky/get-started.html
2. Install prettier as dev dependency
3. Add `.prettierrc` file with the below configurations. 
	```js
	{
		"semi": false,
		"tabWidth": 4,
	    "printWidth": 100,
	    "singleQuote": true,
	    "trailingComma": "none",
	    "bracketSameLine": false,
	}
	```
4. Add dist and node_modules to `.prettierignore` file
5. Run prettier on all files.

**Comment by cdrani** [comment]
@sstephanyy: This might be an even better first issue. Let me know if you want to claim this one instead. I have also updated the `contributing`[https://github.com/cdrani/chorus/blob/develop/contributing.md) file with steps for local development.

**Comment by sstephanyy** [comment]
@cdrani thanks for the contribution guide, will be very helpful. Yes, I wanna claim this one :)

**Comment by sstephanyy** [comment]
@cdrani  sorry to bother again, but I'm facing this error again and again everytime I git commit. Do you have an ideia of what am I doing wrong?
![image](https://github.com/cdrani/chorus/assets/128330097/dade98ef-d6c9-4732-b454-c47181722341)

**Comment by cdrani** [comment]
Solved in #212.
